concourse/tasks/upgrade_extension.yml

```
@@ -0,0 +1,34 @@
+platform: linux
+
+image_resource:
+  type: registry-image
+
+inputs:
+  - name: pxf_package
+  - name: pxf_tarball
+  - name: pxf_src
+  - name: gpdb_package
+  - name: singlecluster
+    optional: true
+  - name: pxf-automation-dependencies
+    optional: true
+
+params:
+  GP_VER:
+  GROUP: pxfCExtensionVersion2
```

```suggestion
  GROUP: pxfExtensionVersion2
```
"Extension" already means C-side extension, a name without C in it is more readable IMO